apps/vc-api/src/vc-api/credentials/credentials.service.spec.ts

```diff
@@ -113,12 +113,10 @@ describe('CredentialsService', () => {
       credential: getChargingDataCredential(did),
       options: issueOptions
     });
-    const presentation = service.presentationFrom(presentationDefinition as IPresentationDefinition, [
-      vc1,
-      vc2
-    ]);
+    const presentation = service.presentationFrom({ credentials: [vc1, vc2] });
+    const presentationDto = JSON.parse(JSON.stringify(presentation));
```

Comment:
Assuming `presentationFrom` remains async, `await` should be added here.
I'm seeing that after adding the `await`, then `presentation` is satisfying the `presentationDto` structure and so this cast can be removed.
```suggestion
    const presentation = await service.presentationFrom({ credentials: [vc1, vc2] });
```